charts/zalenium/templates/ingress.yaml

```
@@ -18,7 +18,7 @@ spec:
   - host: {{ .Values.ingress.hostname }}
     http:
       paths:
-      - path: /
+      - path: /{{ .Values.ingress.contextpath }}
```

I would prefer to have the / part of the injected value if possible and rename the variable to path so it's a one on one match with the actual ingress configuration:

```suggestion
      - path: {{ .Values.ingress.path }}
```

---

Made this change